packages/shopify-app-session-storage-drizzle/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@shopify/shopify-app-session-storage-drizzle",
+  "version": "1.0.0",
+  "description": "Shopify App Session Storage for Drizzle",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Shopify/shopify-app-js.git"
+  },
+  "bugs": {
+    "url": "https://github.com/Shopify/shopify-app-js/issues"
+  },
+  "homepage": "https://github.com/Shopify/shopify-app-js/tree/main/packages/shopify-app-session-storage-drizzle",
+  "author": "Shopify Inc.",
+  "license": "MIT",
+  "main": "./build/cjs/drizzle.js",
+  "types": "./build/ts/drizzle.d.ts",
+  "scripts": {},
+  "publishConfig": {
+    "access": "public"
+  },
+  "keywords": [
+    "shopify",
+    "node",
+    "app",
+    "graphql",
+    "rest",
+    "webhook",
+    "Admin API",
+    "Storefront API",
+    "session storage",
+    "Drizzle"
+  ],
+  "dependencies": {
+    "drizzle-orm": "^0.29.3",
```

Comment:
If we expect folks to also include `drizzle-orm` directly in their apps, we should consider making it a peer dependency instead to prevent multiple instances being installed.